todoman/cli.py

```diff
@@ -317,8 +337,11 @@ def move(ctx, list, ids):
               'Defaults to true.')
 @click.option('--due', default=None, help='Only show tasks due in DUE hours',
               type=int)
+@click.option('--start', default=None, callback=_validate_start_date_param,
+              help='Only shows tasks after given DATE')
```

Comment:
Actually, this doc isn't quite right; it can be before/after a given date. This slipped by before, sorry.